.rubocop.yml

```diff
@@ -48,6 +48,9 @@ Metrics/ModuleLength:
   Exclude:
     - 'spec/**/*' # not a big deal in spec helper module
 
+Style/MixinUsage:
```

Comment:
Rather than set this to 'true', we should just remove the lines that disable it and rely the default (which is to enable it).

Reply:
should I include 
   Exclude:
     - 'spec/**/*' # not a big deal in spec helper module
under each rule ?